src/models/game-media.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface Media {
```

Comment:
```suggestion
export interface GameMedia {
```

Export names should match file names. This provides a better clue as to what is in the file before even opening it.